deployments/container/Makefile

```diff
@@ -28,7 +28,9 @@ IMAGE = $(IMAGE_NAME):$(IMAGE_TAG)
 
 ubuntu22.04: BASE_IMAGE = docker.io/ubuntu:22.04
 
-# Use a generic build target to build the relevant images
+ubi9: BASE_IMAGE = registry.access.redhat.com/ubi9/ubi-micro:latest
```

Comment:
is not good to use latest as it can have random behavior, do you have a static tag?

Reply:
Thanks, pinned to 9.4